urls.py

```diff
@@ -13,7 +13,8 @@
     #url(r'^about/(?P<pagename>[a-z\-]+)/$', 'kaeru.views.about_view'),
     url(r'^documentation/$' , 'kaeru.views.documentation_view'), 
     url(r'^login/$' , 'kaeru.views.login_view'), 
-    url(r'^logout/$' , 'kaeru.views.logout_view'), 
-    url(r'^people/$' , 'kaeru.views.people_view'),
-    url(r'^tutorial/$' , 'kaeru.views.tutorial_view'), 
+    url(r'^logout/$' , 'kaeru.views.logout_view'),
+    url(r'^signup/$' , 'kaeru.views.signup_view'),
+    # url(r'^people/$' , 'kaeru.views.people_view'),
```

Comment:
Why is the people page commented out? Thus makes a test fail that otherwise passes.